src/persist-client/src/impl/machine.rs

```diff
@@ -316,7 +316,7 @@ where
                             .truncate(Instant::now() + FOREVER, &path, self.state.seqno())
                             .await
                     })
-                    .instrument(trace_span!("apply_unbatched_cmd::truncate"))
+                    .instrument(debug_span!("apply_unbatched_cmd::truncate"))
```

Comment:
I might yolo this and a couple other tweaks in myself today :)

Reply:
nevermind, I just added them to this PR XD

Reply:
:bow: